ddtrace/appsec/internal/management.py

```diff
@@ -0,0 +1,67 @@
+"""
+The AppSec module detects security anomalies from inside the application.
+
+Test it with::
+
+    DD_APPSEC_ENABLED=true FLASK_APP=hello.py ddtrace-run flask run
+
+"""
+import logging
+import os.path
+from typing import Any
+from typing import List
+from typing import Mapping
+
+import attr
+
+from ddtrace import Span
+from ddtrace.appsec.internal.protections import BaseProtection
+from ddtrace.utils.formats import get_env
+
+
+log = logging.getLogger(__name__)
```

Comment:
can we use `ddtrace.internal.logger.get_logger` instead?

using our internal logger class will add rate limiting to logs to ensure we don't accidentally spam user's application logs.